assets/prometheus-k8s/rules.yaml

```diff
@@ -1325,17 +1325,41 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: KubeQuotaAlmostFull
```

Comment:
This also introduces a new alert, not just decreases alert severity?

Reply:
Yes, see https://coreos.slack.com/archives/C0VMT03S5/p1599729682146600

Reply:
By the name yes but effectively it's the same alert that existed just the with different names per threshold to account for the behavior that's happening.